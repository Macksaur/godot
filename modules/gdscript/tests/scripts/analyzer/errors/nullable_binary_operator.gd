func test():
    var foo: int? = null
    print(foo + 1) # Should error
