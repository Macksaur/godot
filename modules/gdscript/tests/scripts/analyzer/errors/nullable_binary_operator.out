GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable left-side operand, of type "int?", to be used with the "+" operator.
